source/modules/configuration/index.rst

```diff
@@ -1,13 +1,13 @@
 Configuration
 =============
 
-Le module Configuration permet d'accéder aux options de configuration générale de GLPI : la configuration globale de GLPI, les notifications, les collecteurs, les tâches automatiques, l'authentification, les plugins, les critères d'unicité et les liens externes protocolés.
+Configuration module gives access to all GLPI configuration options: global configuration, notifications, collectors,  automated tasks,authentication, plugins, unicity criteria ans external links.
 
 Les intitulés  ne proposent pas d'actions spécifiques, se reporter aux :doc:`actions communes <../generalites/actions>`.
```

Comment:
I do not really understand this line; I wonder if it should be dropped

Reply:
it's an introduction to the menu and what you have access in it.
Not super usefull, nor useless

Reply:
Surely misplaced then... Well, I won't change that; goal was to rework collectors only